.github/workflows/rust.yml

```diff
@@ -65,18 +77,24 @@ jobs:
           cargo test --tests --examples --verbose ${{ matrix.features }}
       - name: Check wrappers integrity
         # we generate wrappers only without debugmozjs
-        if: ${{ matrix.features == '' }}
+        if: ${{ matrix.features == '--features streams' }}
         run: |
           bash ./mozjs/src/generate_wrappers.sh
           git diff --quiet --exit-code
+      - name: Upload artifact
+        if: ${{ contains(matrix.features, '--features streams') }}
+        uses: actions/upload-artifact@v4
+        with:
+          path: ./target/libmozjs-x86_64-unknown-linux-gnu.tar.gz
+          name: libmozjs-x86_64-unknown-linux-gnu.tar.gz
 
   windows:
     runs-on: windows-latest
     strategy:
       fail-fast: false
       matrix:
-        features: ["--features debugmozjs", ""]
-        target: ["x86_64-pc-windows-msvc", "aarch64-pc-windows-msvc"]
+        features: ["--features debugmozjs", "--features streams"]
+        target: ["", "--target aarch64-pc-windows-msvc"]
```

Comment:
Why we need to add `--target` in here?

Reply:
I hope it can build and test without `--target` if the target is the same as host.
Otherwise, the archive will be created in `target/x86_64-pc-windows-msvc` due to the limitation of [`get_cargo_target_dir`](https://github.com/servo/mozjs/blob/main/mozjs-sys/build.rs#L724)

Reply:
I think we could use `OUT_DIR` for this. Here are result for default than with target invocation:
```
/home/user/Desktop/lol/target/debug/build/lol-bb298d56b00857ef/out
/home/user/Desktop/lol/target/x86_64-unknown-linux-gnu/debug/build/lol-a67d370db0f3eb4c/out
```
so we need to go up (to parrent) four times. Then if current folder is named as rust target (`x86_64-unknown-linux-gnu`) we need to go up one more time. There is also `CARGO_TARGET_DIR` that allows users to override location of target folder (this is used in servo's CI due to space constraints on default storage device (D): https://github.com/servo/servo/blob/main/.github/workflows/windows.yml#L41)

Reply:
I guess I can specify the exact target path, so other parts don't need to be adjusted.
I'll revert back to `target: ["x86_64-pc-windows-msvc", "aarch64-pc-windows-msvc"]` and upload the artifact with the according path.